prereqs_mac.sh

```diff
@@ -3,17 +3,16 @@
 which -s brew
 if [[ $? != 0 ]] ; then
     # Install Homebrew
-    echo "Donwloading and installing homebrew"
+    echo "Downloading and installing homebrew"
     /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 else
     brew update
 fi
 
-# TODO Python checking and installation
-# echo "Donwloading and installing Python3 homebrew"
-# brew install python
+echo "Downloading and installing Python3 using homebrew"
+brew install python3
```

Comment:
Commented this out, because there can be conflicts. So we need to check if python is already installed, then skip this part.